Modules/Test/test/ScoreSettingsTest.php

```diff
@@ -294,7 +299,7 @@ public function testScoreSettingsSectionSummary(): void
                                     <label for="id_3" class="control-label col-sm-4 col-md-3 col-lg-2">tst_reporting_date<span class="asterisk">*</span></label>
                                     <div class="col-sm-8 col-md-9 col-lg-10">
                                         <div class="input-group date il-input-datetime" id="id_3">
-                                            <input type="text" name="" placeholder="YYYY-MM-DD HH:mm" class="form-control form-control-sm" />
+                                            <input type="text" name="" placeholder="YYYY-MM-DD H:mm" class="form-control form-control-sm" />
```

Comment:
I wonder why a `HH` is transformed to a `H` in the UI renderer, but this should not be the problem of te T&A.

Reply:
Interesting, could be related to https://github.com/ILIAS-eLearning/ILIAS/pull/6276